### PR TITLE
Added description to move databases between k8s instances (or namespaces)

### PR DESCRIPTION
**Pull request by markusjnagel**
Please review - added the description how to move a complete set of databases being used for an assessment; moving all tags, stakeholders, reports, assessments, etc. between "Tackle" instances. 

HTH,

cheers,

Markus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -427,10 +427,12 @@ where
 $ ab -n 1000 -c 20 -H 'Accept: application/hal+json' 'http://<host>/controls/business-service?name=service&sort=name&size=1&page=1'
 ```
 
-## Database management
+## Database management {#db-mgmt}
```
Comment (mrizzi) [inline README.md:430]:
```suggestion
## Database management
```

File: docs/MovingTackleDBs.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+### 2) (repeat for every pod) get the DB user
+
+```
+oc exec tackle-application-inventory-postgresql-5f959df4df-zg46k -- printenv POSTGRES_USER
+```
+
+### 3) Export (dump) the database without the flyway tables. 
+As opposed to the development instructions in the [README.md](../README.md#db-mgmt), this uses the `--clean --if-exists` flags, and therefore needs to drop the `-a` flag (data only).
```
Comment (mrizzi) [inline docs/MovingTackleDBs.md:43]:
```suggestion
As opposed to the development instructions in the [README.md](../README.md#database-management), this uses the `--clean --if-exists` flags, and therefore needs to drop the `-a` flag (data only).
```

**Review by mrizzi — COMMENTED** [review]
I've proposed the changes to adopt the built-in anchors based on the header's title.

**Comment by markusjnagel** [comment]
Committed your proposals directly. Thanks, @mrizzi  - TIL :-) markdown minutiae....

File: docs/MovingTackleDBs.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+**THIS WILL OVERWRITE EVERYTHING YOU HAVE IN YOUR DESTINATION TACKLE PODS**
+
+**NOTE** I'm using `oc` here instead of `kubectl` since this is running on a local Red Hat CRC (Code Ready Containers) installation, but you can replace `oc` with `kubctl` - it just happens that this was "hacked" on CRC, not minishift and I didn't have a `kubectl` CLI pointing at my CRC ready.
+
+## Steps to take
+
+### 1) On the source system (or namespace), get your postgres pods (ignore the keycloak-postgresql)
+
+```
+[🎩︎mnagel konveyor]$ oc get pods | grep postgres
```
Comment (mrizzi) [inline docs/MovingTackleDBs.md:28]:
```suggestion
$ oc get pods | grep postgres
```

File: docs/MovingTackleDBs.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+tackle-application-inventory-postgresql-5f959df4df-zg46k   1/1     Running   0          15d
+tackle-controls-postgresql-6bfd5d57ff-cbnfr                1/1     Running   0          15d
+tackle-keycloak-postgresql-9cf6f6879-gbx5w                 1/1     Running   0          15d
+tackle-pathfinder-postgresql-5557cb8bff-vgzng              1/1     Running   0          15d
+[🎩︎mnagel konveyor]$
```
Comment (mrizzi) [inline docs/MovingTackleDBs.md:33]:
```suggestion
$
```

**Review by mrizzi — COMMENTED** [review]
@rromannissen reported this earlier today and I'm keen to get this PR merged ASAP so I filled the comments directly with suggestions.

**Review by mrizzi — APPROVED** [review]
Thanks a lot @markusjnagel for the contribution :+1:

**Pull request merged**